servers/express.js

```diff
@@ -34,20 +57,49 @@ app.all('/*', function (req, res, next) {
 
 app.use(express.static(path.join(__dirname, '..', 'front-end', 'build')));
 
+app.use(bodyParser.json());
+
+app.post('/api/upload', upload.any(), (req, res) => {
+  res.send({files: req.files});
+});
+
+app.post('/api/watchfile', async (req, res) => {
+  const file = path.join(__dirname, '..', 'uploads', req.body.filename);
+  if (!file) {
+    return res.status(404).send({ error: new Error('file not found!')});
+  }
+  pathToLogFile = file;
+  res.status(200).end(collector);
+});
+
+app.get('/api/curr-logfile', (req, res) => {
+  res.status(200).send({currentLogfile: path.parse(pathToLogFile).base});
+});
+
+app.get('/api/timestamps', (req, res) => {
+  res.status(200).send({timestamps: {firstTs, lastTs}});
+});
+
 app.get('/api', function (req, res) {
```

Comment:
maybe this should be `/api/timeseries` now that there are so many other api endpoints?